vfcompositor/Compositor.c

```
@@ -0,0 +1,784 @@
+                    }
+                }
+            }
+            x += FONT_WIDTH;
+        }
+        str++;
+    }
+    window->needs_redraw = true;
+}
+// --- Input Event Handlers ---
+
+void OnMouseMove(int x, int y, int dx, int dy) {
```

Clang (14.0.6)

[warning] 3 adjacent parameters of 'OnMouseMove' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)